specs-go/config.go

```diff
@@ -430,7 +430,7 @@ type LinuxCPU struct {
 // LinuxPids for Linux cgroup 'pids' resource management (Linux 4.3)
 type LinuxPids struct {
 	// Maximum number of PIDs. Default is "no limit".
-	Limit int64 `json:"limit"`
+	Limit *int64 `json:"limit,omitempty"`
```

Comment:
With this changed to pointer, should that mean that it's no longer `REQUIRED`? (i.e., either setting `-1` or omitting means "no limit"?

I wonder how it originally was intended; i.e., if `0` (or "not set", because of Golang's defaults), meaning inherit whatever defaults are present.

Reply:
Yeah, the intention is for it to be the same as most/all of the other cgroup settings we have. As I mentioned in the commit description, we ended up in this situation because of a bunch of unfortunate coincidences and this should correct the situation to be more in line with what most people would expect.